test/spawn-options.spec.js

```diff
@@ -137,11 +137,16 @@ describe('Spawn options', () => {
           }
 
           f.spawn(options, (err, _ipfsd) => {
+            ipfsd = _ipfsd
```

Comment:
I believe this will fail lint - can you run `npm run lint` on the repo and make sure everything passes please?

Reply:
I have and do not see any failures. Were you expecting it to fail because it's assigning a global? ipfsd is defined above, on line 124 so here we are just setting it.

I moved this assignment from below because if any of the below `expect`s failed, it would inadvertently cause the next `it`s to fail.

Reply:
Usually eslint will complain if the first line in the callback is not handling the error, but perhaps its OK since it happens right after `ipfsd  = _ipfsd`. I would still move the assignment below the `expects`s since if there was an error `_ipfsd` would be `null/empty/invalid`.